website/docs/r/managed_disk.html.markdown

```diff
@@ -107,6 +108,8 @@ The following arguments are supported:
 
 * `disk_size_gb` - (Optional, Required for a new managed disk) Specifies the size of the managed disk to create in gigabytes. If `create_option` is `Copy` or `FromImage`, then the value must be equal to or greater than the source's size. The size can only be increased.
 
+* `upload_size_bytes` - (Optional, Required when create_option is `Upload`) Specifies the size of the managed disk to create in bytes. If `create_option` is `Upload`, the value of the disk must be equal to the source disk to be copied in bytes. Source disk size could be calculated with `ls -l` or `wc -c`. More information can be found at [Copy a managed disk](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/disks-upload-vhd-to-managed-disk-cli#copy-a-managed-disk)
```

Comment:
Having this here makes the notes below a little confusing as they pertain to `disk_size_gb`. I suggest moving this above `disk_size_gb`